worlds/aquaria/Regions.py

```diff
@@ -16,7 +16,7 @@
 
 def _has_hot_soup(state:CollectionState, player: int) -> bool:
     """`player` in `state` has the hotsoup item"""
-    return state.has("Hot soup", player)
+    return state.has_any ({"Hot soup", "Hot soup x 2"}, player)
```

Comment:
Does this really work with the space?
```suggestion
    return state.has_any({"Hot soup", "Hot soup x 2"}, player)
```